typed-racket-lib/typed-racket/base-env/base-env.rkt

```diff
@@ -61,7 +61,7 @@
 [equal?/recur (-> Univ Univ (-> Univ Univ Univ) B)]
 [immutable? (asym-pred Univ B (-PS (-is-type 0 (Un -Bytes -BoxTop -String (-Immutable-HT Univ Univ) (-ivec Univ)))
                                    (-not-type 0 (Un (-Immutable-HT Univ Univ) (-ivec Univ)))))]
-[prop:equal+hash -Struct-Type-Property]
+[prop:equal+hash (-struct-property Univ)]
```

Comment:
Should this `prop:equal+hash` type be something like
```
(-lst* (-> -Self -Self (-> Univ Univ B) Univ)
       (-> -Self (-> Univ -Int) -Int)
       (-> -Self (-> Univ -Int) -Int))
```
instead of `Univ` inside the `-struct-property`?

Reply:
Yes. I fixed it in my local branch but I haven't push it yet.